src/sentry/models/release.py

```diff
@@ -92,8 +92,28 @@ def _on_post(project, trigger):
         ):
             schedule_invalidate_project_config(project_id=project.id, trigger=trigger)
 
+    def _subscribe_project_to_alert_rule(project, release, trigger):
+        """
+        TODO: potentially enable custom query_extra to be passed on ReleaseProject creation (on release/deploy)
+        """
+        from django.utils import timezone
+
+        from sentry.incidents.models import AlertRule
+        from sentry.incidents.utils.types import AlertRuleActivationConditionType
+
+        query_extra = f"release:{release.version} AND event.timestamp:>{timezone.now().isoformat()}"
```

Comment:
@wedamija 

Can you validate whether this query makes sense?

also cc @lynnagara or @untitaker ?